docs/platforms/javascript/common/apis.mdx

```diff
@@ -0,0 +1,942 @@
+  callback will be linked to the trace.
+</SdkApi>
+
+<SdkApi
+  name="suppressTracing"
+  signature="function suppressTracing<T>(callback: () => T): T"
+>
+  Ensure that all spans created inside of the provided callback are not sent to
+  Sentry.
+</SdkApi>
+
+<SdkApi
+  name="startNewTrace"
+  signature="function startNewTrace<T>(callback: () => T): T"
+>
+  Start a new trace that is active in the provided callback.
+</SdkApi>
+
+<SdkApi
+  name="startBrowserTracingPageLoadSpan"
+  signature="function startBrowserTracingPageLoadSpan(client: Client, options: StartSpanOptions): Span | undefined"
+  categorySupported={["browser"]}
+>
+  Start an pageload span that will be automatically ended when the page is
+  considered idle. If a pageload/navigation span is currently ongoing, it will
+  automatically be ended first. In most cases, you do not need to call this, as
+  the `browserTracingIntegration` will automatically do that for you. However,
+  if you opt-out of pageload spans, you can use this method to manually start
+  such a span. Please note that this function will do nothing if
+  `browserTracingIntegration` has not been enabled.
+</SdkApi>
+
+<SdkApi
+  name="startBrowserTracingNavigationSpan"
+  signature="function startBrowserTracingNavigationSpan(client: Client, options: StartSpanOptions): Span | undefined"
+  categorySupported={["browser"]}
+>
+  Start an navigation span that will be automatically ended when the page is
+  considered idle. If a pageload/navigation span is currently ongoing, it will
+  automatically be ended first. In most cases, you do not need to call this, as
+  the `browserTracingIntegration` will automatically do that for you. However,
+  if you opt-out of navigation spans, you can use this method to manually start
+  such a span. Please note that this function will do nothing if
+  `browserTracingIntegration` has not been enabled.
+</SdkApi>
+
+<SdkApi
+  name="withActiveSpan"
+  signature="function withActiveSpan<T>(span: Span | null, callback: () => T): T"
+>
+  Runs the provided callback with the given span as the active span. If `null`
+  is provided, the callback will have no active span.
+</SdkApi>
+
+## Tracing Utilities
+
+These utilities can be used for more advanced tracing use cases.
+
+<SdkApi name="spanToJSON" signature="function spanToJSON(span: Span): SpanJSON">
+  Convert a span to a JSON object.
+</SdkApi>
+
+<SdkApi
+  name="updateSpanName"
+  signature="function updateSpanName(span: Span, name: string): void"
+>
+  Update the name of a span. Use this over `span.updateName(name)` to ensure
+  that the span is updated in all backends.
+</SdkApi>
+
+<SdkApi
+  name="getActiveSpan"
+  signature="function getActiveSpan(): Span | undefined"
+>
+  Get the currently active span.
+</SdkApi>
+
+<SdkApi name="getRootSpan" signature="function getRootSpan(span: Span): Span">
+  Get the root span of a span.
+</SdkApi>
+
+## Sessions
+
+<SdkApi name="startSession" signature="function startSession(): void">
```

Comment:
m: I would add a sentence about the session and release health here to avoid confusion with what users would usually think a session is.

Reply:
lol, this shows: we do not have any page in docs about sessions, I think? 😬 This is the closest we have, I guess it's ok... http://localhost:3000/platforms/javascript/guides/react/configuration/releases/#sessions